packages/components/src/presentation-components/instance-filter-builder/MultiTagSelect.tsx

```diff
@@ -101,7 +102,7 @@ function TagLabel({ children, ...props }: any) {
 }
 
 function TagRemove(props: any) {
-  return <components.MultiValueRemove {...props} innerProps={{ ...props.innerProps, className: "iui-button iui-borderless iui-small iui-tag-button" }}>
+  return <components.MultiValueRemove {...props} innerProps={{ ...props.innerProps, className: "iui-button iui-tag-button", ["data-iui-variant"]: "borderless", ["data-iui-size"]: "size" }}>
```

Comment:
I might be wrong, but should the value in `["data-iui-size"]: "size"` be `"small"`?

Reply:
Nice catch, thanks.